tests/acceptance/main.rs

```diff
@@ -7,3 +7,5 @@ mod intercept_global_installs;
 mod notion_current;
 mod notion_deactivate;
 mod notion_pin;
+#[macro_use] // to use the assert_that! macro
+extern crate hamcrest2;
```

Comment:
I know there are warnings being thrown by the compiler on testing, but I believe the Rust 2018 idiom is to _not_ use `extern crate` or `#[macro_use]` and instead directly import the specific macro (in this case, `assert_that`). @chriskrycho Can you confirm?

Reply:
Oh right rust 2018 - I'll change this back to stop using `#[macro_use]` and `extern crate`